app/models/formModels.scala

```diff
@@ -9,6 +9,19 @@ import play.api.data.validation.{Constraint, Invalid, Valid, ValidationError}
 
 object formModels {
 
+  final case class AddUserToGroupFormData(email: String)
+
+  object AddUserToGroupFormData {
+
+    val form: Form[AddUserToGroupFormData] =
+      Form(
+        mapping("email" -> nonEmptyText)(AddUserToGroupFormData.apply)(
```

Comment:
après test, c'est pas super d'avoir une erreur si on a mis une espace
```suggestion
        mapping("email" -> nonEmptyText)(email => AddUserToGroupFormData(email.trim))(
```